srcpkgs/pillow-simd/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'pillow-simd'
+pkgname=pillow-simd
+version=5.3.0.post1
+revision=1
+archs="x86_64*"
+build_style=python3-module
+pycompile_module="PIL"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel libjpeg-turbo-devel libopenjpeg2-devel
+ zlib-devel tiff-devel freetype-devel lcms2-devel libwebp-devel libimagequant-devel freetype-devel harfbuzz-devel fribidi-devel libraqm-devel"
```

Comment:
this line is too long, please add line breaks at aproximately 80 characters.

Reply:
okay, I've made the necessary changes.